### PR TITLE
ozone adapter - fixup for gdpr and device objects

### PR DESCRIPTION
**Pull request by afsheenb**
## Type of change
- [X ] Bugfix

## Description of change

Updating ozone adapter to send device w/h and GDPR consent string in request.

- engineering@ozoneproject.com

### CODE REVIEW

**Comment by afsheenb** [comment]
@bretg we've submitted updated unit tests, please review and let us know if any further updates are needed.

cheers,

--afsheen

File: modules/ozoneBidAdapter.js [hunk 1]
```diff
@@ -77,13 +78,18 @@ export const spec = {
 
     delete ozoneRequest.test; // don't allow test to be set in the config - ONLY use $_GET['pbjs_debug']
     if (bidderRequest.gdprConsent) {
+      utils.logInfo('ADDING GDPR');
```
Comment (bretg) [inline modules/ozoneBidAdapter.js]: These debug log entries are likely to be confusing to others - please consider removing them or at least annotating with your bidder name so it's more clear what's adding GDPR.

File: modules/ozoneBidAdapter.js [hunk 2]
```diff
@@ -77,13 +78,18 @@ export const spec = {
...
       ozoneRequest.regs = {};
       ozoneRequest.regs.ext = {};
       ozoneRequest.regs.ext.gdpr = bidderRequest.gdprConsent.gdprApplies === true ? 1 : 0;
       if (ozoneRequest.regs.ext.gdpr) {
-        ozoneRequest.regs.ext.consent = bidderRequest.gdprConsent.consentString;
+        ozoneRequest.user = {};
+        ozoneRequest.user.ext = {'consent': bidderRequest.gdprConsent.consentString};
       }
+    } else {
+      utils.logInfo('WILL NOT ADD GDPR');
```
Comment (bretg) [inline modules/ozoneBidAdapter.js]: same - please either remove or add your bidder name

**Comment by afsheenb** [comment]
@bretg done, we've explicitly added the bidder name to the debug logging statements for clarity.

**Pull request merged**